server/app.js

```diff
@@ -1,6 +1,9 @@
 import express from 'express';
 import logger from 'morgan';
 import bodyParser from 'body-parser';
+import Routes from './routes';
+
+require('dotenv').config();
```

Comment:
You should be able to use import statement to include the 'dotenv' module, instead of using require. Not sure but I think the requirement document stated use of ES6 import and export statements ?